2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -214,7 +262,11 @@ public void getElementByNegativeIndex() {
     @Test
     @Order(20)
     public void getElementByIndexEqualToListSize() {
```

Comment:
@erneman1 let's remove this method to `getElementByIndexThrowsExceptionWhenIndexIsOutOfBound`